Automattic/jetpack
# Moving jetpack-sync-module to a psr-4 class.

Parent Issue #12751

#### Changes proposed in this Pull Request:

This PR should have no functional changes. All sync modules should function exactly the same as before

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* If you're an Automattician, include a shortlink to the p2 discussion with Jetpack Product here.

#### Testing instructions:







* Try this branch out on a Jurassic.ninja site or on your docker testing site.
* Connect the site and activate the recommended features.
* Point your testing site to your .com sandbox to observe sync actions in real time.
* Try a full sync and ensure there are no php errors in your site's error logs
* Try some actions on your site, and ensure incremental sync is working.

#### Proposed changelog entry for your changes:

* None
